### ISSUE TITLE
Warnings about Grad strides do not match bucket view strides.

### ISSUE THREAD

**Issue by togekiss0603** [facebookresearch/vggt#339]
Thank you for your solid work! 
When I train vggt from scratch I encouter the following warnings（not errors）：

/home/qzy/vggt/vggt/models/vggt.py:65: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  with torch.cuda.amp.autocast(enabled=False):
/home/qzy/anaconda3/envs/open3d/lib/python3.11/site-packages/torch/autograd/graph.py:825: UserWarning: Grad strides do not match bucket view strides. This may indicate grad was not created according to the gradient layout contract, or that the param's strides changed since DDP was constructed.  This is not an error, but may impair performance.
grad.sizes() = [2, 32, 1, 1], strides() = [32, 1, 32, 32]
bucket_view.sizes() = [2, 32, 1, 1], strides() = [32, 1, 1, 1] (Triggered internally at /opt/conda/conda-bld/pytorch_1729647429097/work/torch/csrc/distributed/c10d/reducer.cpp:327.)
  return Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass

Is this normal? Will it affect the model's performance?
I use gloo instead of NCCL because it‘s leading to something wrong like Using network Socket.

**Comment by jytime** [comment]
Hi it will not affect the model's accuracy, but may slow down you training (should be very slight I think)

**Comment by togekiss0603** [comment]
> Hi it will not affect the model's accuracy, but may slow down you training (should be very slight I think)

Thanks for your answer. And i'd like to know if i want to train a vggt model **from scratch**, how much size and scale of dataset should i prepare (e.g. only co3Dv2 OK?), and how much GB on GPUs?
